### PR TITLE
Replace choice field literals with references

### PR DESCRIPTION
**Pull request by ddabble**

### CODE REVIEW

File: checkin/models.py [hunk 1]
```diff
@@ -30,14 +30,18 @@ def __str__(self):
...
-    level_choices = (
-        (1, "Nybegynner"),
-        (2, "Viderekommen"),
-        (3, "Ekspert"),
+    BEGINNER = 1
+    EXPERIENCED = 2
+    EXPERT = 3
+
+    LEVEL_CHOICES = (
+        (BEGINNER, "Nybegynner"),
```
Comment (zootos) [inline checkin/models.py]: It would be nice to have translations of these, probably fits the theme and changes in this PR to do it here.

Comment (ddabble) [reply checkin/models.py]: Yes, I agree 🙂

**Comment by ddabble** [comment]
Yeah, I realized just after pushing 😅

**Comment by zootos** [comment]
No worries, just making sure everything is good :)

**Pull request merged**